### DISCUSSION TITLE
Can't reconnect (no response) to Homekit when using Bridges

### DISCUSSION THREAD

**Discussion by lileffects** [HomeSpan/HomeSpan#943] [General]
So when I use example 1 or any sketch that doesn't use bridges I don't have any issues with reconnection to Homekit after powering off/on or resetting the board. However, if I have a sketch that uses Bridges it will only work after initially pairing to Homekit, but gets stuck with a "No Response" if I power off/on or reset the board. 

Any ideas? I'm trying to control 4 relays and anything 3 or more forces the use of bridges. Using two relays works as expected.

**Comment by HomeSpan** [comment]
Are you using a HomeKit Hub, such as an Apple TV or Home Pod?

**Comment by lileffects** [comment]
No. Just the HomeKit app and the ESP32-C3.

**Reply by HomeSpan** [reply]
Use of HomeSpan without a Home Hub is not supported as it is unpredictable.  Apple started requiring Home Hub a few years ago for full HomeKit connectivity and functionality.  It may be that without a Home Hub certain sketches without a Bridge will work, but they will not operate as a Bridge.  Apple has not provided any detailed docs on what works and what doesn't work when not using a Home Hub. 

Did you try running one of the examples that have a Bridge to ensure there is not an issue with your specific sketch?

**Comment by lileffects** [comment]
As long as I keep it to two or less devices it works. Once you have three
or more it forces the use of a bridge even if you don’t call it out in the
sketch.

So an Apple TV works as a home hub?
